ajax/libs/angular-payments/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "angular-payments",
+  "filename": "angular-payments.min.js",
+  "description": "Directive for formatting and validating credit card forms",
+  "keywords": [
+    "stripe",
+    "stripe.js",
+    "stripejs",
+    "stripe-angular",
+    "stripeAngular",
+    "angular-payments",
+    "angularPayments"
+  ],
+  "version": "1.0.7",
+  "author": {
+    "name": "Lauri Hynynen",
+    "url": "https://github.com/laurihy"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/zumper/angular-payments.git"
+  },
+  "license": "MIT",
+  "npmName": "angular-payments",
```

Comment:
There is only one version on NPM but 8 on GitHub. It's better to use git auto-update config.